pennylane_lightning/src/simulator/Measures.hpp

```diff
@@ -309,6 +310,98 @@ class Measures {
         return expected_value_list;
     };
 
+    /**
+     * @brief Complete a single Metropolis-Hastings step.
+     *
+     * @param sv state vector
+     * @param tk User-defined functor for producing transitions
+     * between metropolis states.
+     * @param gen Random number generator.
+     * @param distrib Random number distribution.
+     * @param s1 initial state
```

Comment:
I assume `s1` is the basis state index? Or something else? I'd prefer a more descriptive name for this value that is tied to its type and utility.

Reply:
good point!